frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
I couldn't reproduce this with Google Chrome. Which browser were you using? Did you have to revert the whole pull request or would it have been sufficient to reintroduce the `JSON.parse(JSON.stringify(geojson))`?